chunky/src/java/se/llbit/chunky/chunk/BlockPalette.java

```diff
@@ -338,11 +338,14 @@ public static Map<String, Consumer<Block>> getDefaultMaterialProperties() {
     materialProperties.put("minecraft:redstone_torch", block -> {
       block.emittance = 1.0f;
     });
+    materialProperties.put("minecraft:redstone_wall_torch", block -> {
+      block.emittance = 1.0f;
+    });
     materialProperties.put("minecraft:torch", block -> {
-      block.emittance = 50.0f;
+      block.emittance = 5.0f;
```

Comment:
:+1: for reducing the high brightness.

The emittance values are "normalized" currently, such that a Minecraft emittance value of 15 is 1.0 in Chunky for a 16x16x16 (area = 1536) units block (plus some rounding to get "nice" values for no reason).The torch is 2x2x10, i.e. has an area of 88 units. By that calculation, the torch should have an emittance of 17.45 in Chunky.

If you only take the high-emittance (i.e. yellow) part of the torch into account, that area is roughly 20, so an emittance value of 50 would actually be too low (should be 76.8).

Same considerations for the other torches. Since these changes affect every render created with future Chunky versions (unless the materials are tweaked manually), we should be careful about them.

Reply:
We noticed that torches outshone outshone other emitters, even though the emittance values are normalized, and even though the torch emittance value is techincally too low. How can that be?

Reply:
Actually, torches have an in-game light level of 14, so the calculation turns out to be 71.68.

Reply:
Undo for now and wait for community feedback?

Reply:
Yes, maybe a post on Reddit could be made, with some comparisons.

Reply:
Torches and sould torches have been reset to their default values.